packages/hooks/src/usePaymentAgentTransferVisible.ts

```diff
@@ -0,0 +1,17 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const usePaymentAgentTransferVisible = () => {
+    const { client } = useStore();
+    const { is_authorize } = client;
```

Comment:
added `is_authorize` variable to client_store, in order to be sure that our "get_settings" request will be fired only after authorize request.